pytorch_lightning/trainer/trainer.py

```diff
@@ -130,6 +131,7 @@ def __init__(
         devices: Optional[Union[List[int], str, int]] = None,
         gpus: Optional[Union[List[int], str, int]] = None,
         auto_select_gpus: bool = False,
+        hpus: Optional[int] = None,
```

Comment:
@jerome-habana Would you be ok with not having a separate flag for this and integrate this with `devices` and `accelerator`?

We are currently in the process of reducing and re-evaluating the trainer arguments as they've grown quite large.

Reply:
@justusschock has this been decided? We still have the IPU flag for example. is there any issue to track this development?

Reply:
> Yes, we are currently depreciating those flags for readability.

We haven't decided yet that we will be deprecating those flags.

Reply:
what is the final decision ? Shall I proceed with the existing change ?